### DISCUSSION TITLE
APPEUI, DEVEUI, APPKEY ADAFRUIT FEATHER 32U4

### DISCUSSION THREAD

**Discussion by arlx07** [Azure/iotedge-lorawan-starterkit#1583] [General]
Cum as putea gasi APPEUI, DEVEUI sau APPkey pentru placile Adafruit Feather 32u4?

**Comment by danigian** [comment]
Hello @arlx07,
please have a look at the official Adafruit website for any question related to that specific hardware.

[Here](https://learn.adafruit.com/adafruit-feather-32u4-radio-with-lora-radio-module/using-the-rfm-9x-radio) you can find a sample for your device.

**Comment by arlx07** [comment]
I looked at the link above, but I want to know these details that I mentioned. Do not specify APPEUI, DEVEUI or APPKEY.

    On Thursday, March 3, 2022, 10:41:31 AM GMT+2, Daniele Antonio Maggio ***@***.***> wrote:  
 
 


Hello @arlx07,
please have a look at the official Adafruit website for any question related to that specific hardware.

Here you can find a sample for your device.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by danigian** [reply]
You could try to have a look [at this repository](https://github.com/mcci-catena/arduino-lmic)
In the examples folder I can see some "feather" related samples that might work with your device.

**Comment by arlx07** [comment]
Doresc sa trimit date de la aceste placi , mai exact umiditate si temperatura, catre live-objects. Aveti un cod pentru senzorul DHT22 si pentru a trimite date catre platforma in timp real? Va multumesc !
